File: src/main/java/com/proximyst/schizoparadise/scheduling/EffectTicking.java
package com.proximyst.schizoparadise.scheduling;

import com.proximyst.schizoparadise.Paradise;
import com.proximyst.schizoparadise.data.SchizophrenicPlayer;
import lombok.RequiredArgsConstructor;
import org.bukkit.Bukkit;
import org.bukkit.entity.Player;
import org.bukkit.scheduler.BukkitRunnable;

@RequiredArgsConstructor
public class EffectTicking extends BukkitRunnable {
    private final Paradise main;

    @Override
    public void run() {
        for (Player player : Bukkit.getOnlinePlayers()) {
            SchizophrenicPlayer.getPlayer(player).trySymptom(main);
        }
    }
}

File: src/main/kotlin/com/proximyst/schizoparadise/schedule/EffectTicking.kt
package com.proximyst.schizoparadise.schedule

import com.proximyst.schizoparadise.Main
import com.proximyst.schizoparadise.data.SchizophrenicPlayer
import org.bukkit.Bukkit
import org.bukkit.scheduler.BukkitRunnable

class EffectTicking(private val main : Main) : BukkitRunnable() {
  override fun run() {
    Bukkit.getOnlinePlayers().forEach { SchizophrenicPlayer.new(it).trySymptom(main) }
  }
}